docs/articles/rtl.md

```diff
@@ -0,0 +1,53 @@
+All Nebular components were updated to support RTL out of the box.
+
+Now every component, that previously could be positioned via some setting, besides old `left` and `right` options, also support logical `start` and `end` properties, similar to flexbox. Value of `start` and `end` depends on current layout direction. For LTR it's left, for RTL - right.
```

Comment:
The components that accept a position as a setting now also support logical `start` and `end` values, similar to flexbox. Value of `start` and `end` depends on current layout direction. For LTR it's left and for RTL - right.
For instance, if we need the sidebar to be positioned logically depending on a language direction, then instead of setting it to `left` we can set its position to `start`:
```
<nb-sidebar start></nb-sidebar>
```

Document direction could be set through the `NbThemeModule.forRoot` call.  Supported values are 'ltr' and 'rtl'.